src/main/java/com/unfamilia/eggbot/domain/raidpackage/ItemCategory.java

```diff
@@ -1,8 +1,13 @@
 package com.unfamilia.eggbot.domain.raidpackage;
 
-public enum ItemCategory {
-    POTION,
-    ENHANCEMENT,
-    FLASK,
-    ENCHANT
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name = "itemCategory")
+public class ItemCategory extends PanacheEntity {
```

Comment:
Why is this enum considered an entity?

Reply:
It doesn't need to be, I was on the fence about it - thought perhaps it would be better to encapsulate each level as an entity but actually unless we identify some information linked to an ItemCategory we want to store (I can't think of anything off the top of my head) there's no reason for it.

Reply:
https://en.wikipedia.org/wiki/KISS_principle
https://www.baeldung.com/jpa-persisting-enums-in-jpa